docs/scarpet/language/LoopsAndHigherOrderFunctions.md

```diff
@@ -49,7 +49,7 @@ for(range(1000000,1100000),check_prime(_))  => 7216
 
 From which we can learn that there is 7216 primes between 1M and 1.1M
 
-### `while(cond, limit)`
+### `while(cond, limit,expr)`
```

Comment:
To maintain consistency, it's a good idea to put spaces after commas in function arguments.
```suggestion
### `while(cond, limit, expr)`
```